spring-boot-docs/src/main/asciidoc/spring-boot-features.adoc

```diff
@@ -2185,17 +2184,15 @@ to the home page at "/" and keep the default for everything else:
 
 [source,java,indent=0]
 ----
-	@Configuration
+	@EnableWebSecurity
```

Comment:
This is already not true because this `WebSecurityConfigurerAdapter` will be invoked (rather than the ones configure by Spring Boot) regardless of if `@EnableWebSecurity`.

Admittedly, it does give you slightly different results since the ignored patterns are no longer used. However, the intent is to [address](http://docs.spring.io/spring-security/site/docs/4.0.x/reference/htmlsingle/#hello-web-security-java-configuration):

> The name of the configureGlobal method is not important. However, it is important to only configure AuthenticationManagerBuilder in a class annotated with either @EnableWebSecurity, @EnableGlobalMethodSecurity, or @EnableGlobalAuthentication. Doing otherwise has unpredictable results.

The reason for this is because we need to ensure any classes that are configuring `AuthenticationManagerBuilder` are instantiated before we build the `AuthenticationManager`. In practice, this is done using `EnableGlobalAuthenticationAutowiredConfigurer` which eagerly initializes beans that are annotated (or meta-annotated) with `@EnableGlobalAuthentication`.

These instructions are likely a big part of the reason users experience ordering issues with Spring Security and Spring Boot.